doc/modules/model_evaluation.rst

```diff
@@ -806,6 +810,18 @@ precision-recall curve as follows.
         :scale: 75
```

Comment:
I am thinking that we should document the function returning the ratios.
I think that we should move the confusion matrix presentation before the accuracy score that use the TP FP TN FR already. Like this we could document the ratio function just after the confusion matrix.

Reply:
Attempted.